server/src/gql/resolver.py

```diff
@@ -72,3 +72,65 @@ def resolve_add_new_mates(
     family.mate_ids += [new_mate.id for new_mate in new_mates]
     family.save()
     return new_mates
+
+
+@mutation.field("assignMates")
+def resolve_assign_mates(
```

Comment:
could we wrap it in ErrorBounded and return a (AppError, List[str]) instead?

Reply:
Resolver functions are caught via Ariadne. If any of the errors are raised, they are sent back with the appropriate error code. I tried with ErrorBounded before, and it messes up all the GQL stuff.